modules/tfm/tfm/boards/partition/flash_layout.h

```diff
@@ -53,8 +53,10 @@
  * IMAGE_SECONDARY, SCRATCH is used as a temporary storage during image
  * swapping.
  */
+#ifdef CONFIG_TFM_BL2
```

Comment:
@oyvindronningstad , how could this be done to match the upstream flash layout header? CONFIG_TFM_BL2 does not exist in upstream. But there is the BL2 cmake variable, right?

Reply:
`#ifdef BL2` should work.

Reply:
This needs to be upstreamed in either case, so i think it's perfectly fine this way now until an upmerge brings in the eventual upstream change